apps/site/src/components/NavBar/NavBar.module.scss

```diff
@@ -3,73 +3,74 @@
 
 .nav {
 	width: 100%;
-	filter: drop-shadow(0px 2.5px 2px rgba(0, 0, 0, 0.4));
-
 	@include utils.media-breakpoint-up(sm) {
-		position: sticky;
+		position: fixed;
+		z-index: 1;
+	}
+
+	@media only screen and (max-width: 991.5px) {
+		background-color: rgba(20, 20, 20, 0.7);
 		z-index: 101;
-		top: 0;
+		position: fixed;
+		text-align: right;
+		backdrop-filter: blur(7px);
 	}
+
+	background-color: transparent;
+	position: relative;
+	color: white;
 }
 
 .navbar {
-	background-image: url("~@/assets/images/lined_paper.svg");
-	background-size: cover;
-	background-position: left bottom;
-	background-repeat: no-repeat;
+	position: absolute;
+	top: 0;
 }
 
 .logo {
 	position: absolute;
-	left: 30px;
-	width: 20vw;
-	height: 20vw;
-	text-align: center;
-	transform: rotate(-16.81deg) translateY(-5vw);
+	width: 4vw;
+	height: 4vw;
+	top: 1vh;
+
+	@media only screen and (max-width: 991.5px) {
+		position: relative;
+		width: 50px;
+		height: 50px;
+		align-items: left;
+		text-align: left;
+		top: 0;
+	}
 }
 
 .text {
 	margin-left: 1vw;
 	margin-right: 1vw;
-	font-size: 2rem;
-	font-weight: bold;
+	font-weight: 600;
+	font-size: 1rem;
 	text-decoration: none;
+	@media only screen and (max-width: 991.5px) {
+		margin-top: 2vw;
+		margin-bottom: 2vw;
+		font-size: 1.5rem;
+	}
```

Comment:
Suggestion [repeating]: this block of css can be refactored with Bootstrap's mixin `media-breakpoint-up(lg)`.